tests/ci/run_benchmark_build_tests.sh

```diff
@@ -26,7 +26,8 @@ function build_aws_lc_fips {
       -DENABLE_DILITHIUM=ON \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DBUILD_SHARED_LIBS=1 \
-      -DBUILD_TESTING=OFF
+      -DBUILD_TESTING=OFF \
+      -DENABLE_FIPS_ENTROPY_CPU_JITTER=1
```

Comment:
NP: Will this negatively affect any of our benchmarks? Should we separately benchmark builds both w/ and w/o CPU jitter entropy?

Reply:
This is just a test build to make sure it can build, the actually canary doesn't run with this flag.